EIPS/eip-1959.md

```diff
@@ -0,0 +1,79 @@
+---
+eip: 1959
+title: Method to check if a chainID is part of the history of chainIDs
```

Comment:
Sorry to make this comment now and not yesterday: I think the title should reflect this change is about introducing an opcode to EVM.

Reply:
No worries, thanks for reviewing the PRs!